hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/ssl/SSLFactory.java

```diff
@@ -83,6 +81,13 @@
   public static final String SSL_SERVER_EXCLUDE_CIPHER_LIST =
       "ssl.server.exclude.cipher.list";
 
+  public static final String LOCALIZED_PASSWD_FILE_NAME = "hops.ssl.keystores.passwd.name";
+  public static final String DEFAULT_LOCALIZED_PASSWD_FILE_NAME = "material_passwd";
+  public static final String LOCALIZED_KEYSTORE_FILE_NAME = "hops.ssl.keystore.name";
+  public static final String DEFAULT_LOCALIZED_KEYSTORE_FILE_NAME = "k_certificate";
+  public static final String LOCALIZED_TRUSTSTORE_FILE_NAME = "hops.ssl.trustore.name";
+  public static final String DEFAULT_LOCALIZED_TRUSTSTORE_FILE_NAME = "t_certificate";
```

Comment:
I suggest you change the name of the configuration variables to `*_KEY` e.x. `LOCALIZED_PASSWD_FILE_NAME_KEY`, so there would be no misunderstanding if it's the actual file name - as it was until this PR - or if it's a configuration property.

Reply:
I changed that.